kibana/kibana.yml

```diff
@@ -1,67 +1,79 @@
 # Kibana is served by a back end server. This controls which port to use.
-port: 5601
+# server.port: 5601
```

Comment:
I noticed a few other places where we either grep/look for 5601.  If we allow it to change, that seems to be a problem with 'the check'